open_spiel/python/algorithms/policy_aggregator.py

```diff
@@ -228,13 +246,10 @@ def assert_type(cond, msg):
       if pid == turn_player:
         # update the current node
         # will need the observation to query the policies
-        if state not in self._policy:
+        if state_key not in self._policy:
```

Comment:
Is this fixing a bug that existed in the code previously?

Reply:
Yes I believe this should be state_key instead of state. Because state is a pyspiel.state object which doesn't seem right to be a key.